packages/emotion-primitives/src/index.js

```diff
@@ -0,0 +1,50 @@
+import reactPrimitives from 'react-primitives'
+
+import { createEmotionPrimitive } from './createEmotion'
+import { splitProps } from './splitProps'
+
+const primitives = ['Text', 'View', 'Image']
+
+const assignPrimitives = styled => {
+  Object.assign(
+    styled,
+    primitives.reduce((getters, alias) => {
+      getters[alias.toLowerCase()] = styled(reactPrimitives[alias])
+      return getters
+    }, {})
+  )
+
+  Object.assign(
+    styled,
+    primitives.reduce((getters, alias) => {
+      const tag = alias.toLowerCase()
+      getters[alias] = styled[tag]()
+      getters[alias].tag = reactPrimitives[alias]
+      getters[alias].displayName = `emotion.${tag}`
+      return getters
+    }, {})
+  )
+
+  return styled
+}
+
+const emotion = createEmotionPrimitive(splitProps)
```

Comment:
will it be possible to `createEmotionPrimitive` with other `splitProps` implementation? are there use cases for this?

Reply:
Use case is -

To split component props, style props and style overrides so that we can forward them without logging any warnings.